test/utils.couchbase.test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+require('should');
+const { isCouchbaseVersionSupported } = require('../utils/couchbase.js');
+
+describe('Couchbase Utils', () => {
+    describe('isCouchbaseVersionSupported', () => {
```

Comment:
not sure why the indent goes with 4 spaces here, in the other places it's 2 spaces.

Reply:
@xavierchow will check the linting